docs/tools_user_guide.md

```diff
@@ -293,61 +293,43 @@ You can see the options available for any command by running `benchit COMMAND --
 
 The `benchmark` command supports the arguments from [Devices and Runtimes](#devices-and-runtimes), as well as:
 
-### Input Script
+### Input Scripts
 
-Name of the script (.py) file, within the search directory, to be benchmarked.
+Name of the script (.py) files to be benchmarked.
 
 Usage: 
-- `benchit INPUT_SCRIPT` 
-- `benchit benchmark INPUT_SCRIPT`
+- `benchit INPUT_SCRIPTS` 
```

Comment:
`INPUT_SCRIPT` has references all over the tutorials and other docs. Make sure to clean all of those up. Repo should have zero instances of "INPUT_SCRIPT" now.

Reply:
All references to INPUT_SCRIPT have been replaced by INPUT_SCRIPTS